Source/DafnyStandardLibraries/src/Std/JSON/ZeroCopy/Serializer.dfy

```diff
@@ -304,7 +306,11 @@ module Std.JSON.ZeroCopy.Serializer {
     var wr := Members(obj, wr);
     var wr := StructuralView(obj.r, wr);
     Seq.LemmaConcatIsAssociative2(writer.Bytes(), Spec.Structural<View>(obj.l, Spec.View), Spec.ConcatBytes(obj.data, Spec.Member), Spec.Structural<View>(obj.r, Spec.View));
-    assert wr.Bytes() == writer.Bytes() + Spec.Bracketed(obj, Spec.Member);
+
+    assert wr.Bytes() == writer.Bytes() + Spec.Bracketed(obj, Spec.Member) by {
+      hide *;
```

Comment:
This is surprising. How does that work?

Reply:
I'm not sure, and actually it might be good to understand first. I'll work on a feature that will show what is hidden/revealed when using `*`.